diff -ruN odyssey-pre-jit-clean/Source/JavaScriptCore/assembler/CMakeLists.txt odyssey-125-jit-test-3/Source/JavaScriptCore/assembler/CMakeLists.txt
--- JavaScriptCore.orig/assembler/CMakeLists.txt	2018-07-13 21:32:32.568181234 +0000
+++ JavaScriptCore/assembler/CMakeLists.txt	2018-07-09 07:50:56.875919286 +0000
@@ -1,4 +1,8 @@
 list(APPEND JSC_SRC
 #	    assembler/MacroAssemblerX86Common.cpp
+	assembler/LinkBuffer.cpp
+	assembler/MacroAssembler.cpp
+	assembler/MacroAssemblerPPC.cpp
+	assembler/PPCAssembler.cpp
 )
 
diff -ruN odyssey-pre-jit-clean/Source/JavaScriptCore/jit/CMakeLists.txt odyssey-125-jit-test-3/Source/JavaScriptCore/jit/CMakeLists.txt
--- JavaScriptCore.orig/jit/CMakeLists.txt	2018-07-13 21:32:32.580180776 +0000
+++ JavaScriptCore/jit/CMakeLists.txt	2018-07-09 07:51:25.527746813 +0000
@@ -3,6 +3,7 @@
     jit/ClosureCallStubRoutine.cpp
     jit/ExecutableAllocator.cpp
     jit/ExecutableAllocatorFixedVMPool.cpp
+    jit/GCAwareJITStubRoutine.cpp
     jit/HostCallReturnValue.cpp
     jit/JIT.cpp
     jit/JITCode.cpp
